package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@asyncapi/optimizer",
+  "version": "0.0.1",
+  "description": "This library will optimize the AsyncAPI specification file.",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "scripts": {
+    "test": "echo \"No test specified\"",
+    "build": "tsc",
+    "lint": "eslint --max-warnings 0 --config .eslintrc .",
+    "get:version": "echo $npm_package_version",
+    "get:name": "echo $npm_package_name",
+    "generate:assets": "npm run build",
+    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/asyncapi/optimizer.git"
+  },
+  "author": "Khuda Dad Nomani",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/asyncapi/optimizer/issues"
+  },
+  "homepage": "https://github.com/asyncapi/optimizer#readme",
+  "devDependencies": {
+    "@semantic-release/commit-analyzer": "^8.0.1",
+    "@semantic-release/github": "^7.2.3",
+    "@semantic-release/npm": "^7.1.3",
+    "@types/jest": "^26.0.23",
+    "@types/node": "^15.12.1",
+    "jest": "^27.0.4",
+    "typescript": "^4.3.2"
+  }
```

We should also add release related config

```suggestion
  },
  "publishConfig": {
    "access": "public"
  },
  "release": {
    "branches": [
      "master"
    ],
    "plugins": [
        [
          "@semantic-release/commit-analyzer",
            {
              "preset": "conventionalcommits"
            }
        ],
        [
          "@semantic-release/release-notes-generator",
          {
            "preset": "conventionalcommits"
          }
        ],
        "@semantic-release/npm",
        "@semantic-release/github"
    ]
  }
```